pyk4a/pyk4a.py

```diff
@@ -14,7 +14,9 @@ class Result(Enum):
 
 
 class K4AException(Exception):
-    pass
+    def __init__(self, errorMessage=None):
```

Comment:
I am curious, is there a reason for this? Exception class already prints what you input it. `raise Exception("will show this message")`

Reply:
oh! I think I didn't realize because I used to, in other projects built my own exceptions classes which had more than just the error message. I can remove that, no problem!